migrations/20240103140930-parcelles-table.js

```diff
@@ -0,0 +1,264 @@
+'use strict'
+
+let dbm
+let type
+let seed
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate
+  type = dbm.dataType
+  seed = seedLink
+}
+
+exports.up = async function (db) {
+  await db.createTable('cartobio_parcelles', {
+    record_id: {
+      type: 'uuid',
+      notNull: true,
+      primaryKey: true
+    },
+    id: {
+      type: 'string',
+      notNull: true,
+      primaryKey: true
+    },
+    geometry: {
+      type: 'geometry(GEOMETRY, 4326)',
+      notNull: true
+    },
+    commune: {
+      type: 'string'
+    },
+    cultures: {
+      type: 'jsonb',
+      notNull: true
+    },
+    conversion_niveau: {
+      type: 'string'
+    },
+    engagement_date: {
+      type: 'date'
+    },
+    commentaire: {
+      type: 'string'
+    },
+    annotations: {
+      type: 'jsonb'
+    },
+    created: {
+      type: 'datetime'
+    },
+    updated: {
+      type: 'datetime'
+    },
```

Comment:
```suggestion
    created: {
      type: 'datetime',
      timezone: true
    },
    updated: {
      type: 'datetime',
      timezone: true
    },
```

Tu penses que ça vaut le coup de stocker l'info ? Ou s'il faut l'ajouter, on le fait en sortie d'API ?

Reply:
Stocker l'info est pertinent si on veut gérer plusieurs fuseaux en front, c'est à dire si on veut afficher "dernière modification 13h heure de Paris" à quelqu'un en Guyane parce que la dernière modification a été faite depuis Paris.

Hors j'ai plutôt l'impression que si quelqu'un est en Guyane, on va afficher toutes les dates en heure guyanaise.
Et dans ce cas on a plutôt intérêt à enregistrer tout en UTC et convertir à l'affichage. Mais bonne remarque, car je ne suis pas sûre qu'on convertisse en UTC dans la migration pour le moment :)